19_hidden_message.html

```diff
@@ -36,20 +36,25 @@
   <body>
     <div id="hidden-message">This is our hidden message!</div>
     <button id="button-show">Show secret message</button>
+    <button id="button-hide">Hide secret message</button>
   </body>
 
   <script>
   // Get the element #hidden-message and assign it to hiddenMessage variable
   const hiddenMessage = document.querySelector('#hidden-message')
-  
+
   // Get the element #button-show and assign it to buttonShow variable
   const buttonShow = document.querySelector('#button-show')
+  const buttonHide = document.querySelector('#button-hide')
 
   // Listen for a buttonShow click
   buttonShow.onclick = function () {
     // buttonShow has been clicked, show the hidden message.
     hiddenMessage.style.display = 'block';
   }
+  buttonHide.onclick = function () {
+    hiddenMessage.style.display = "";
```

Comment:
awesome!

Nitpicking:
- You could also set this to `none`, but the CSS is helping you out.
- Stick to single quotes or double quotes :)